react/src/components/ServiceLauncherModal.tsx

```diff
@@ -151,27 +176,93 @@ const ServiceLauncherModal: React.FC<ServiceLauncherProps> = ({
     },
   });
 
+  const [
+    commitModifyEndpoint,
+    // inInFlightCommitModifyEndpoint
+  ] = useMutation<ServiceLauncherModalModifyMutation>(graphql`
+    mutation ServiceLauncherModalModifyMutation(
+      $endpoint_id: UUID!
+      $props: ModifyEndpointInput!
+    ) {
+      modify_endpoint(endpoint_id: $endpoint_id, props: $props) {
+        ok
+        msg
+      }
```

Comment:
(This feedback is related to core's GraphQL Schema, not WebUI)
The return type of `modfy_endpoint` GraphQL query needs to be changed. It should return `Endpoint` type.

Reply:
I just updated query in server-side but somehow generated graphql file didn't change. I need to dig it more...
https://github.com/lablup/backend.ai/pull/1859/files#diff-09158c3c58d5e1ee85d64c91e1805be9cdbcc9ba856492cb8a8e759760a9fd5dR667

Reply:
Resolved!